implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/v1/DynatraceExporterV1Test.java

```diff
@@ -40,9 +43,10 @@
  *
  * @author Johnny Lim
  */
-class DynatraceMeterRegistryTest {
+class DynatraceExporterV1Test {
 
     private final DynatraceMeterRegistry meterRegistry = createMeterRegistry();
+    private final DynatraceExporterV1 meterRegistryImpl = createMeterRegistryImpl();
```

Comment:
Could you please remove "impl" from the names (methods, variables)? The exporter is not a registry anymore.